opm/output/data/Aquifer.hpp

```diff
@@ -66,6 +66,42 @@ namespace Opm { namespace data {
             }
             return 0.;
         }
+
+        template <class MessageBufferType>
```

Comment:
Do we have a "formal specification" of the `MessageBufferType` API - so that this can be tested from opm-common?

Reply:
Well, you have to look in dune for that. This is not new, it's there for wells, cells, groups and guide rates.

Reply:
I know - but it is new in opm-common; and although this is a small thing which can be merged I am weary of a situation where code can not be tested because an actual class needed for instantiation is not present. 

I am also pondering this due to my [own sin](https://github.com/OPM/opm-common/blob/master/opm/common/utility/Serializer.hpp) which I guess should follow the Dune `MessageBufferType` api.

Reply:
[MessageBufferIf](https://dune-project.org/doxygen/2.7.1/classDune_1_1MessageBufferIF.html). Maybe we should add a comment?

Reply:
Comment added.

Reply:
> but it is new in opm-common;

Strictly speaking, that's not entirely true.  We already have something similar for [`data::Wells`](https://github.com/OPM/opm-common/blob/debfc70450e4fd9e2a0576340b18c6d402272081/opm/output/data/Wells.hpp#L264-L267) and other 
closely related input/output types.  At least some of this is tested in [`test_Summary`](https://github.com/OPM/opm-common/blob/debfc70450e4fd9e2a0576340b18c6d402272081/tests/test_Summary.cpp#L1705-L1709) with a custom [`MessageBuffer`](https://github.com/OPM/opm-common/blob/debfc70450e4fd9e2a0576340b18c6d402272081/tests/test_Summary.cpp#L1664) type.

I don't think the current testing is sufficient since it mostly acts as a check that the custom `MessageBuffer` doesn't alter any of the bits passed to it, but it is nevertheless better than nothing at all in my opinion.